backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/button/BpkButton.kt

```diff
@@ -131,7 +130,7 @@ fun BpkButton(
                 ButtonText(text)
```

Comment:
does it also need to be applied here or is this one correct? if so maybe the modifier should be applied internally to `ButtonText`

Reply:
hmm in that shouldn't be a problem as for this case the icon is always at the start and it will take its allocated space first then the text will come after
![Screenshot 2024-05-14 at 16 34 09](https://github.com/Skyscanner/backpack-android/assets/19841948/cde39363-391d-4f78-8cd9-c84c8890578c)

Reply:
also can you add a snapshot test example that covers these cases to make sure it is covered by a test

Reply:
I wasn't sure if we need one, but sure let me add that 😄